Foundation/URLSession/NativeProtocol.swift

```diff
@@ -0,0 +1,27 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+
+
+import CoreFoundation
+import Dispatch
+
+internal class _NativeProtocol: URLProtocol {
+    
+    override class func canonicalRequest(for request: URLRequest) -> URLRequest {
+        return request
+    }
+
+    override func startLoading() {
+       NSRequiresConcreteImplementation()
```

Comment:
Minor nit: In your next PR, could you make sure to fix the indentation here and in the method below? They should be indented by four spaces, not three.

Reply:
Sure thanks ! Actually, the concrete implementation of StartLoading will be moved here  from ```HTTPURLProtocol```. This is just a placeholder  for now ..